docs-site/content/ko/worktree/guide.md

```
@@ -465,35 +396,30 @@ flowchart TD
+계획은 메인 체크아웃에서 진행합니다.
+
 ```bash
 # Terminal 1에서
-> /moai plan "사용자 인증 시스템 구현" --worktree
+> /moai plan "사용자 인증 시스템 구현"
 ```
 
-**출력**:
+**출력 (예시)**:
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)